go.mod

```diff
@@ -1,8 +1,10 @@
 module github.com/google/wire
 
 require (
+	github.com/fatih/color v1.7.0
```

Comment:
Fly-by: this is adding to the dependency load of wire, which is being imported by clients. Maybe we should consider separate modules for the wire *library* and wire *command*  - the library should have very minimal dependencies (none?)

Reply:
@zombiezen 	WDYT? We could split `wire` into two modules....

Reply:
-1 here. While separating into separate modules works well for the Go CDK libraries, we actually stand to gain something here by keeping the code generator binary in the same module. Once modules is on by default, we can more easily ensure that the Wire code generator matches the runtime support library that is in use (see #160).